### PR TITLE
Untar WDL

### PR DESCRIPTION
**Pull request by benjamincarlin**

### CODE REVIEW

**Comment by benjamincarlin** [comment]
@danxmoran it would be nice to switch to wdl 1.0 so we....

don't have to call separate tasks to get file size from the gsutil paths

can use nested scatters and scatter over a list containing paths to tarball lists

File: NeMO/src/main/wdl/untar/UntarFiles.wdl [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+         # copy to bucket
+         gsutil -m cp *~{file_extension} ~{gsutil_output_path}
+    >>>
+
+    runtime {
+        docker: "google/cloud-sdk:slim"
+        # if the input size is less than 1 GB adjust to min input size of 1 GB
+        disks: "local-disk " + ceil(1 * (if tarball_size < 1 then 1 else tarball_size)) + " HDD"
+        cpu: 1
+        memory: "3.5 GB"
```
Comment (danxmoran) [inline NeMO/src/main/wdl/untar/UntarFiles.wdl]: You can put `preemptible: 3` here to get cheaper disks.

Comment (danxmoran) [reply NeMO/src/main/wdl/untar/UntarFiles.wdl]: And tweak memory to be < 3.5 gig for papi v2.

File: NeMO/src/main/wdl/untar/UntarFiles.wdl [hunk 2]
```diff
@@ -0,0 +1,56 @@
+version 1.0
+
+workflow UntarFiles {
+    input {
+        Array[File] tarball_lists
```
Comment (danxmoran) [inline NeMO/src/main/wdl/untar/UntarFiles.wdl]: Why not point to the tarballs directly? Array too big?

File: NeMO/src/main/wdl/untar/UntarFiles.wdl (hunk 1 above)
Comment (benjamincarlin) [reply NeMO/src/main/wdl/untar/UntarFiles.wdl]: I have preemptible in the options json, should I put it hear as well?

**Pull request merged**